common/src/main/java/io/druid/metadata/PasswordProvider.java

```diff
@@ -34,5 +34,15 @@
 })
 public interface PasswordProvider
```

Comment:
We also have internal implementations for `PasswordProvider` that do things like run blob storage object through cloud KMS decryption.